src/Commands/VersionCommand.ts

```diff
@@ -29,7 +29,35 @@ export class VersionCommand implements CommandModule {
     });
   }
 
-  public async handler(): Promise<void> {
+  private static getGlobalNpmVersion(globalNpmList: string) {
+    const globalMatches: RegExpMatchArray | null = globalNpmList.match(
+      / @neuralegion\/bright-cli@(.*)\n/
+    );
+    const globalNpmVersion: string = (
+      globalMatches && globalMatches[1] ? globalMatches[1] : ''
+    )
+      .replace(/"invalid"/gi, '')
+      .trim();
+
+    return globalNpmVersion;
+  }
+
+  private static getLegacyGlobalNpmVersion(globalNpmList: string) {
+    const legacyGlobalMatches: RegExpMatchArray | null = globalNpmList.match(
+      / @neuralegion\/nexploit-cli@(.*)\n/
+    );
+    const legacyGlobalNpmVersion: string = (
+      legacyGlobalMatches && legacyGlobalMatches[1]
+        ? legacyGlobalMatches[1]
+        : ''
+    )
+      .replace(/"invalid"/gi, '')
+      .trim();
+
+    return legacyGlobalNpmVersion;
+  }
```

Comment:
Eliminate any irrelevant changes that have no relation to the task you are currently working on. PR must stay atomic